.Documentation/articles/innoactive-creator/localization.md

```diff
@@ -0,0 +1,43 @@
+# Localization
+
+Localization can be used to allow courses to run in different languages. Available Languages can be selected during runtime by using the training menu.
+
+![How to change language during runtime](../images/localization/menu-languages.png)
+
+## Where is localization used?
+
+Mainly localization is used for text to speech audio during the training courses. But it can also be used to choose which already existing audio file should be played and for specific customization for the desktop mode.
```

Comment:
```suggestion
Localization is mainly used for text to speech audio during a training course. But it can also be used to choose which already existing audio file should be played and for specific customization for the desktop mode.
```